h5py/_objects.templ.pyx

```diff
@@ -111,15 +111,15 @@ if hasattr(os, "register_at_fork"):
 # time is taking actions that may create or invalidate identifiers. See the
 # "locking code" section above.
 #
-# See also __cinit__ and __dealloc__ for class ObjectID.
+# See also __cinit__ and __del__ for class ObjectID.
 
 import gc
 import weakref
 import warnings
 
 # Will map id(obj) -> weakref(obj), where obj is an ObjectID instance.
 # Objects are added only via ObjectID.__cinit__, and removed only by
-# ObjectID.__dealloc__.
+# ObjectID.__del__.
```

Comment:
I am concerned about changing from `__dealloc__` to `__del__`.

Reply:
Can you elaborate a little bit more what the concern is?